roles/replace_node/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Create tmp dir for storing peer data
+- name: Create temporary storage directory
+  tempfile:
+    state: directory
+    suffix: _peer
+  register: tmpdir
+  delegate_to: 127.0.0.1
+  run_once: True
+
+# Set the glusterd location
+- name: Set the path of glusterd.info file
+  set_fact:
+    glusterd_libdir: "/var/lib/glusterd"
+    peer_tmp_dir: "{{ tmpdir['path'] }}"
+
```

Comment:
The RHHI specific work is not added yet. As I mentioned in the review comments:

This does not include hyperconverged usecase yet.
However, it is not much work from here, it is about adding:

```
# Switch host into maintenance mode:
- ovirt_hosts:
    state: maintenance
    name: "{{ gluster_maintenance_old_node }}"
```

In the beginning. I'll do soon, this is the generic brick movement. I have requested some answers from Gobinda to do that.

* The old node should not be accessible by hostname, since the new node will be accessed by the hostname.
* The hostname resolution to the hostname has to be turned off, and should be accessible only ip or maybe another hostname.